data/human/culture conversations.txt

```diff
@@ -163,6 +192,34 @@ mission "Class Trip to Port"
 
 
 
+mission "Blind Landing"
+	minor
+	landing
+	source
+		government "Free Worlds"
+		attributes "farming"
+		not attributes "station" "frontier"
+	to offer
+		"year" > "bumper crop year" + 2
+		random < 1
+		has "Trash Fire Crops: declined"
+	on offer
+		conversation
+			`Smog fills your windscreen as you make your descent, trying and failing to see anything more than a few metres in front of you. The autopilot systems show solid green across the full range of indicators though, and traffic control has confirmed a lock.`
+			choice
+				`	(Land on autopilot.)`
+				`	(Return to orbit.)`
+					goto orbit
+			`	You let the craft perform the landing, staring blankly at the haze until it clears.`
+			`	On the ground, you see the cause, fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```

Comment:
```suggestion
			`	On the ground, you see the cause: fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```
Not really a fan of the narrative here. As I see it, there are two possibilities:
1) This is planned and the folks would have proper permits for controlled burn. If so, then why are they having to argue with firefighters about putting it out?
2) The farmers just decided to do this without telling any government authority/the fire dept. If so, I think there's a much larger problem than just people arguing with firefighters. This seems like a pretty large hazard, and as you say below, it makes the air bad for months.
Also, if this is something happening every few years, why is this conflict happening at all? Surely the firefighters and farmers would be aware of what's going on instead of arguing to each other as the event goes down.
Basically, there's too little information here for me to reasonably explain what `<first>` is witnessing. A few extra sentences would be helpful.

Reply:
This one is based off the Haze situation in Singapore years ago. Back then, plantation workers in Indonesia would burn their crops for I don't remember reasons, and it caused visibility and air quality to drop for a good while.
I haven't seen the haze for years, but this story is intended to be told of illegal slash and burning.

Reply:
Gotcha. Then yeah, I think some elaboration would be good. I'm really not sure what the best direction would be to go with it, but I think the idea would be for there to be more going on between the farmers and firefighters, potentially involving authorities.

Reply:
Maybe more people on the ground? I don't want to involve the player into the whole political situation and just an observer far away.

Reply:
```suggestion
			`	On the ground, you see the cause, fields upon fields of plantations burning. You see a crowd blocking off a smaller group of firefighters, arguing and trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```Test Case 1 of trying a rewrite.

Reply:
```suggestion
			`	On the ground, you see the cause, fields upon fields of plantations burning. You see a crowd of mostly farmers blocking off a smaller group of firefighters, arguing and trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```
Maybe mentioning that the crowd is made of farmers helps clarify the burn's intentional. And I get where you're coming from: looking back at this a couple weeks later, it's a little unreasonable to try and explain the entire context to this situation within the story.

Reply:
Yee, the crowd of farmers would be better. 
My alternative take would be to show that they're farmers than tell the reader they're what they are, but this is one of those points where I'd throw in the towel and let some future writer figure it out and edit.